tests/containers/bci_prepare.pm

```diff
@@ -27,22 +27,28 @@ use testapi;
 
 
 sub packages_to_install {
-    my $host_version = get_required_var('HOST_VERSION');
-    $host_version =~ s/-SP/./;
+    my ($version, $sp, $host_distri) = get_os_release;
     my $arch = get_required_var('ARCH');
-
     # common packages
-    my @packages = ('git-core', 'python3', 'python3-devel', 'gcc');
-    if ($host_version eq "12.5") {
-        push @packages, 'python36-pip';
-    } elsif ($host_version eq '15') {
-        assert_script_run("SUSEConnect -p PackageHub/15/$arch");
-        push @packages, ('go1.10', 'skopeo');
-    } elsif ($host_version =~ /15\./) {
-        # Desktop module is needed for SDK module, which is required for installing go
-        assert_script_run("SUSEConnect -p sle-module-desktop-applications/$host_version/$arch");
-        assert_script_run("SUSEConnect -p sle-module-development-tools/$host_version/$arch");
-        push @packages, ('go', 'skopeo');
+    my @packages = ('git-core', 'python3', 'gcc');
+    if ($host_distri eq 'ubuntu') {
+        push @packages, ('python3-dev', 'python3-pip', 'golang');
+    } elsif ($host_distri eq 'centos') {
+        push @packages, ('python3-devel', 'python3-pip', 'golang');
+    } elsif ($host_distri eq 'sles') {
+        my $version = "$version.$sp";
+        push @packages, 'python3-devel';
+        if ($version eq "12.5") {
+            push @packages, 'python36-pip';
+        } elsif ($version eq '15.0') {
+            assert_script_run("SUSEConnect -p PackageHub/15/$arch");
```

Comment:
```suggestion
            # On SLES15 go needs to be installed from packagehub. On later SLES it comes from the SDK module
            assert_script_run("SUSEConnect -p PackageHub/15/$arch");
```

I'd add a short comment to avoid confusion in the future.